app/lib/core/view_models/signin_view_model.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:app/core/enums/view_state.dart';
+import 'package:app/core/services/authentication/auth_service.dart';
+import 'package:app/core/services/navigation/nav_service.dart';
+import 'package:app/core/services/service_locator.dart';
+import 'package:app/core/view_models/base_view_model.dart';
+import 'package:app/ui/views/home/home.dart';
+import 'package:flutter/material.dart';
+
+class SignInViewModel extends BaseViewModel {
+  //Inject auth service
+  final AuthenticationService _authenticationService =
```

Comment:
Again, these services as singletons make some sense because we may need the service across the app (and we don't want to hand around our single service nor do we want to be spinning up new instances all the time - unlike the widgets where we should have some static factory but we don't need or want the same instances all the time)